pkg/fixtures/test/panic_on_no_return_value_test.go

```diff
@@ -0,0 +1,23 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	mocks "github.com/vektra/mockery/v2/mocks/github.com/vektra/mockery/v2/pkg/fixtures"
+)
+
+func TestPanicOnNoReturnValue(t *testing.T) {
+	m := mocks.NewPanicOnNoReturnValue(t)
+	m.EXPECT().DoSomething()
+
+	defer func() {
+		if r := recover(); r != nil {
+			assert.NotNil(t, r)
```

Comment:
This needs to be `require.NotNil` so that the test will fail immediately. Otherwise the `r.(string)` cast would panic.